src/systems/ec/ec-component/ec-component-banner/ec-component-banner.scss

```diff
@@ -3,6 +3,16 @@
  * @define banner
  */
 
+// Import base and generic
+@import '@ecl/ec-base/ec-base';
 @import '@ecl/generic-component-banner/generic-component-banner';
 
-@include ecl-banner();
+// Check if overridden dependencies are already loaded, if needed
+@include check-imports(
+  ('ec-style-typography-heading', 'ec-style-typography-paragraph')
```

Comment:
Missing check on `ecl-image` (`ec-style-image`i guess)

Reply:
I don't see icons in banner